services/web-app/components/glossary/glossary-list/glossary-list.module.scss

```
@@ -0,0 +1,48 @@
+@use '@carbon/react/scss/spacing' as spacing;
+@use '@carbon/react/scss/type' as type;
+@use '@carbon/react/scss/theme' as theme;
+@use '@carbon/react/scss/utilities/convert' as convert;
+@use '@carbon/react/scss/breakpoint' as breakpoint;
+
+.glossary-entry {
+  padding-top: convert.rem(36px);
```

```suggestion
  padding-top: convert.rem(36px);
  color: theme.$text-primary;
```

minor, if we add this is works for all themes.